tests/integration/profiling/rule_based_profiler/test_profiler_basic_workflows.py

```diff
@@ -326,7 +326,7 @@ def test_profiler_save_and_load(data_context_with_taxi_data):
                         "_amount",
                     ],
                 },
-                "variables": {},
```

Comment:
I wonder what caused this?

Reply:
hmm I'm trying to dig into it some more - seems weird